ui/components/app/app-header/app-header.stories.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import AppHeader from '.';
 
+// eslint-disable-next-line import/no-anonymous-default-export
```

Comment:
Is this required? This rule should already be disabled for `*.stories.js` files.

```suggestion
```

Reply:
Updated in [17344](https://github.com/MetaMask/metamask-extension/pull/17344)